app/herms.hs

```diff
@@ -185,11 +185,11 @@ export targets format = do
 
 getServingsAndConv :: Int -> String -> Config -> (Maybe Int, Conversion)
 getServingsAndConv serv convName config = (servings, conv)
-  where servings = case serv of
-                   0 -> case defaultServingSize' config of
-                           0 -> Nothing
-                           j -> Just j
-                   i -> Just i
+  where servings = if serv <= 0
+                    then case defaultServingSize' config of
```

Comment:
Can you un-indent the `then` and `else` here?

Another way to approach this nested conditional would be to use `MultiWayIf`

Reply:
Well spotted! Thanks for the pointers, I'm new to Haskell and need every piece of advice I can get. I'll use `MultiWayIf` instead